meta-security-smack/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -15,6 +15,21 @@ _ System -wx---
 ^ System::Run rwxat-
 ^ System rwxa--
 EOF
+
+    mkdir -p ${D}/${libdir}/tmpfiles.d
+    cat >${D}/${libdir}/tmpfiles.d/packet-forwarding.conf <<EOF
+t /proc/sys/net/ipv4/conf/all/forwarding - - - - security.SMACK64=*
+t /proc/sys/net/ipv6/conf/all/forwarding - - - - security.SMACK64=*
+t /proc/sys/net/ipv4/conf/default/forwarding - - - - security.SMACK64=*
+t /proc/sys/net/ipv6/conf/default/forwarding - - - - security.SMACK64=*
+EOF
+
+    mkdir -p ${D}/${base_libdir}/udev/rules.d
+    cat >${D}/${base_libdir}/udev/rules.d/85-netdev-ipconf-smacklabel.rules <<EOF
+SUBSYSTEM=="net", ENV{ID_NET_NAME}=="", RUN+="/bin/sh -c '/usr/bin/chsmack -a \* /proc/sys/net/ipv4/conf/%k/*'", RUN+="/bin/sh -c '/usr/bin/chsmack -a \* /proc/sys/net/ipv6/conf/%k/*'"
```

Comment:
Ah, did you mean the usage of shell in the bbapend ? I though you meant the udev rule.

Shell usage here is just for consistency with the pre-existing bbappend. I would have chosen chosen  dropping files with static content under files and then append them to SRC_URI. However, for reasons beyond my understanding, whoever rolled the original bbappend decided to cat the file in place with a shell here-document. I think it is ugly and it does not make much sense because the generated content is fully static but I wanted to keep the bbappend consistently with the pre-existing style. The other alternative would be to convert everything to file entries appended to SRC_URI.